src/views/pacbio/PacbioSamples.vue

```diff
@@ -150,7 +150,7 @@ export default {
   methods: {
     async provider() {
       try {
-        await this.setRequests()
+        this.items = await this.setRequests()
```

Comment:
Don't think you need `this.items` as this.setRequests() doesnt return anything
Can also remove items from data